### PR TITLE
CUPS yaml update

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Add description for `tc`  plugin.

##### Test Plan

1. Take a look in CI and grammar.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? documentation, integration and dashboard.
- Can they see the change or is it an under the hood? If they can see it, where? no
- How is the user impacted by the change? a plugin well documented.
- What are there any benefits of the change? A more clear information for our users.
</details>

### CODE REVIEW

File: collectors/cups.plugin/metadata.yaml [hunk 1]
```diff
@@ -1,53 +1,66 @@
...
-    method_description: ''
+    metrics_description: "Monitor CUPS performance for achieving optimal printing system operations. Monitor job statuses, queue lengths, and error rates to ensure smooth printing tasks."
+    method_description: "The plugin uses CUPS shared library to connect and monitor the server."
   supported_platforms:
     include: []
     exclude: []
   multi_instance: true
   additional_permissions:
-    description: ''
+    description: "The plugin needs to access the server. Netdata sets permissions during installation time to reach the server trough its library."
```
Comment (shyamvalsan) [inline collectors/cups.plugin/metadata.yaml:28]:
```suggestion
    description: "The plugin needs to access the server. Netdata sets permissions during installation time to reach the server through its library."
```

File: collectors/cups.plugin/metadata.yaml [hunk 2]
```diff
@@ -1,53 +1,66 @@
...
-      description: ''
+      description: ""
     performance_impact:
-      description: ''
+      description: ""
 setup:
   prerequisites:
-    list: []
+    list:
+      - title: Minimum setup
```
Comment (shyamvalsan) [inline collectors/cups.plugin/metadata.yaml:39]: Do we really need this? title can be empty I think

File: collectors/cups.plugin/metadata.yaml [hunk 3]
```diff
@@ -1,53 +1,66 @@
...
+        title: "Config options"
         enabled: true
-      list: []
+      list:
+        - name: update every
+          description: Data collection frequency.
+          default_value: 1
+          required: false
+        - name: command options
+          description: Additinal parameters for collector
```
Comment (shyamvalsan) [inline collectors/cups.plugin/metadata.yaml:57]:
```suggestion
          description: Additional parameters for the collector
```

File: collectors/cups.plugin/metadata.yaml [hunk 4]
```diff
@@ -1,53 +1,66 @@
...
+        title: "Config options"
         enabled: true
-      list: []
+      list:
+        - name: update every
+          description: Data collection frequency.
+          default_value: 1
+          required: false
+        - name: command options
+          description: Additional parameters for the collector
```
Comment (shyamvalsan) [inline collectors/cups.plugin/metadata.yaml:57]: What does this mean? Can we give an example?

**Review by shyamvalsan — APPROVED** [review]

**Review by sashwathn — APPROVED** [review]

File: collectors/cups.plugin/metadata.yaml (hunk 2 above)
Comment (thiagoftsm) [reply collectors/cups.plugin/metadata.yaml:39]: VScode reports error, I am not sure about overall `yamllint`, ping @Ancairon

File: collectors/cups.plugin/metadata.yaml (hunk 4 above)
Comment (thiagoftsm) [reply collectors/cups.plugin/metadata.yaml:57]: This is a generic description. This collector does not have other parameters.

**Pull request merged**